study-builder/studies/testboston/emails/adhoc_completed.en.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Email looks something like this:

![Screen Shot 2020-11-25 at 3 10 29 PM](https://user-images.githubusercontent.com/5713833/100277719-a13d8a80-2f31-11eb-9d2f-6c989bafdfe1.png)